<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KreuzstichGUI::AboutDialog</class>
 <widget class="QDialog" name="KreuzstichGUI::AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Kreuzstich</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="logo_lbl">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/icons/thimble.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="kreuzstichVersionString_lbl">
         <property name="text">
          <string>(kreuzstichVersionString)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="libkreuzstichVersionString_lbl">
         <property name="text">
          <string>(libkreuzstichVersionString)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="copyright_lbl">
         <property name="text">
          <string>(copyright)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="using_lbl">
     <property name="text">
      <string>Using:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="qtVersionString_lbl">
     <property name="text">
      <string>(QtVersionString)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="MagickVersionString_lbl">
     <property name="text">
      <string>(MagickVersionString)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="BoostVersionString_lbl">
     <property name="text">
      <string>(BoostVersionString)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="xml2VersionString_lbl">
     <property name="text">
      <string>(xml2VersionString)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>KreuzstichGUI::AboutDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
